src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/CollectionPropertiesShouldBeReadOnlyTests.cs

```diff
@@ -431,5 +431,39 @@ class A
 }
 ");
         }
+
+        [Fact, WorkItem(4461, "https://github.com/dotnet/roslyn-analyzers/issues/4461")]
+        public async Task CA2227_CSharp_InitPropertyRecord()
+        {
+            await new VerifyCS.Test
+            {
+                LanguageVersion = CodeAnalysis.CSharp.LanguageVersion.CSharp9,
```

Comment:
This *might* work, @Evangelink 

```suggestion
                LanguageVersion = CodeAnalysis.CSharp.LanguageVersion.CSharp9,
                ReferenceAssemblies = ReferenceAssemblies.Net.Net50,
```

Reply:
Of course it will work! Totally forgot about that, thanks!